set.mm

```diff
@@ -27081,21 +27068,39 @@ generally appear in a single form (either definitional, but more often
     drnfc1.1 $e |- ( A. x x = y -> A = B ) $.
     $( Formula-building lemma for use with the Distinctor Reduction Theorem.
        Usage of this theorem is discouraged because it depends on ~ ax-13 .
-       (Contributed by Mario Carneiro, 8-Oct-2016.)  Avoid ~ ax-11 .  (Revised
-       by Wolf Lammen, 10-May-2023.)  (New usage is discouraged.) $)
+       (Contributed by Mario Carneiro, 8-Oct-2016.)  Avoid ~ ax-8 , ~ ax-11 .
+       (Revised by Wolf Lammen, 22-Sep-2024.)  (New usage is discouraged.) $)
     drnfc1 $p |- ( A. x x = y -> ( F/_ x A <-> F/_ y B ) ) $=
+      ( vw weq wal cv wcel wnf wnfc wceq wb eleq2w2 drnf1 albidv df-nfc 3bitr4g
+      syl ) ABGAHZFIZCJZAKZFHUBDJZBKZFHACLBDLUAUDUFFUCUEABUACDMUCUENEFCDOTPQAFC
+      RBFDRS $.
+    $( $j usage 'drnfc1' avoids 'ax-8' 'ax-11'; $)
+
+    $( Obsolete version of ~ drnfc1 as of 22-Sep-2024.  (Contributed by Mario
+       Carneiro, 8-Oct-2016.)  Avoid ~ ax-11 .  (Revised by Wolf Lammen,
+       10-May-2023.)  (New usage is discouraged.)
+       (Proof modification is discouraged.) $)
+    drnfc1OLD $p |- ( A. x x = y -> ( F/_ x A <-> F/_ y B ) ) $=
       ( vw weq wal cv wcel wnf wnfc eleq2d drnf1 albidv df-nfc 3bitr4g ) ABGAHZ
       FIZCJZAKZFHSDJZBKZFHACLBDLRUAUCFTUBABRCDSEMNOAFCPBFDPQ $.
-    $( $j usage 'drnfc1' avoids 'ax-11'; $)
 
     $( Formula-building lemma for use with the Distinctor Reduction Theorem.
        Proof revision is marked as discouraged because the minimizer replaces
        ~ albidv with ~ dral2 , leading to a one byte longer proof.  However
        feel free to manually edit it according to conventions.  Usage of this
```

Comment:
I think the part 

> "Proof revision is marked as discouraged because the minimizer replaces ~ albidv with ~ dral2 , leading to a one byte longer proof.  However feel free to manually edit it according to conventions."

can be removed, because ~dral2 is marked with "New usage is discouraged", so the minimizer will not replace ~albidv with ~dral2.

Reply:
It will not under the current conditions.  But dral2 is discouraged, because it depends on ax-13.  This mechanism was invented as a quick and dirty fix to constrain ax-13 to a limited range of mathematics.  Should we ever mark these theorems otherwise, the problem will be there again.

Can we keep this topic and discussions related to it out of this pull request please?

Reply:
> Can we keep this topic and discussions related to it out of this pull request please?
OK, is there an issue for this topic, or do you keep track of it?

Reply:
I added a TODO, that is about what I am willing to do in this pull request.